dev-vcs/svn2cl/svn2cl-0.14.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Create a GNU-style ChangeLog from subversion's svn log --xml output"
+HOMEPAGE="https://arthurdejong.org/svn2cl/"
+SRC_URI="${HOMEPAGE}/${PN}/${P}.tar.gz"
```

Comment:
Don't use HOMEPAGE in SRC_URI, HOMEPAGE is meant to be multi-value.

Reply:
Ah, fixed.